astropy/io/misc/asdf/schemas/astropy.org/astropy/units/equivalency-1.0.0.yaml

```diff
@@ -0,0 +1,38 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://astropy.org/schemas/astropy/units/equivalency-1.0.0"
+tag: "tag:astropy.org:astropy/units/equivalency-1.0.0"
+
+title: |
+  Represents unit equivalency.
+
+description: |
+  Supports serialization of equivalencies between units
+  in certain contexts
+
+definitions:
+  equivalency:
+    type: object
+    properties:
+      name:
+        type: string
+      kwargs_names:
+        type: array
+        items:
+          type: string
+      kwargs_values:
+        type: array
+        items:
+          anyOf:
+            - $ref: "tag:stsci.edu:asdf/unit/quantity-1.1.0"
+            - type: number
+            - type: "null"
+
+type: object
+properties:
+  equivalencies:
```

Comment:
This level of nesting seems redundant, but I wonder if it is required for validation? In other words, could the top-level object just be an `array` here? And then in the `to_tree` method below, you would just return a list rather than a dictionary with a single list item.

Reply:
I am not sure. Can you point to an example where the top level object is an array?

Reply:
I thought this isn't allowed but it works so I must be wrong. I just pushed a commit with these changes.